### DISCUSSION TITLE
如何禁用lsposed对模块的自动更新？

### DISCUSSION THREAD

**Discussion by CcLtM** [LSPosed/LSPosed#1520] [Q&A] [answered]
如题，我安装了一个模块，但作者表示在之后的版本会逐渐停止对安卓11的支持。但我不想更新手机系统（害怕丢root，还有怕root后丢失而校准过的指纹传感器会不会又不能用了）。我的手机是一加9R。

**Comment by yujincheng08** [comment, marked as answer]
我们哪有自动更新模块的功能。。。

**Reply by CcLtM** [reply]
> 我们哪有自动更新模块的功能。。。

可是Lsposed隔段时间就提示我某某模块已经更新，请强行停止该模块，可我最近一段时间都没有动过这个模块
